src/chapter3/slurm.md

```diff
@@ -1 +1,68 @@
 # Job batching & SLURM
+
+Launching and running jobs on M3 is controlled by [SLURM](https://slurm.schedmd.com/). You don't really need to know a lot about it in order to use it, so this section will take you through the basics of what you will need for what we are doing.
+
+If you want a complete guide on SLURM in M3, you can find it [here](https://docs.massive.org.au/M3/slurm/slurm-overview.html).
+
+## Submitting simple jobs
+
+As we discussed in the previous section we use bash scripts to run jobs on M3. We can submit these jobs using the `sbatch` command. For example, if we have a bash script called `hello.sh` that contains the following:
+
+```bash
+#!/bin/bash
+
+#SBATCH --ntasks=1
+#SBATCH --mem=1MB
+#SBATCH --time=0-00:01:00
+#SBATCH --job-name=hello
+#SBATCH --partition=m3i
+#SBATCH --mail-user=jmar0066@student.monash.edu
+#SBATCH --mail-type=BEGIN,END,FAIL
+
+echo "Hello World"
+```
+
+We can submit this job using the following command:
+
+`sbatch hello.sh`
+
+This will submit the job to the queue, and you will get an email when the job starts, finishes, or fails. You can also check the status of your job using the `squeue` command.
+
+## Options
+
+You might have noticed the `#SBATCH` lines in the bash script. These are called options, and they tell SLURM how to run the job. The options we used in the example above are:
+
+- `ntasks`: The number of tasks or processes to run.
+- `mem`: The amount of memory to allocate to the job.
+- `time`: The maximum amount of time the job can run for.
+- `job-name`: The name of the job. Up to 15 characters.
+- `partition`: The partition to run the job on.
+- `mail-user`: The email address to send job status emails to.
+- `mail-type`: The types of emails to send.
+
+There are a lot more options that you can use, and you can find a more complete list [here](https://docs.massive.org.au/M3/slurm/simple-batch-jobs.html).
+
+In particular, if you want to run multithreading or multiprocessing jobs, or you need a gpu, there are more options you need to configure.
+
+## Interactive jobs
```

Comment:
Really cool to see this and well explained. I didn't even know this was possible.